superset-frontend/src/common/components/common.stories.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+// import { withKnobs, select, boolean, text } from '@storybook/addon-knobs';
+import Modal from './Modal';
+import Tabs from './Tabs';
+
+export default {
+  title: 'Common Components',
+  // decorators: [withKnobs],
+};
+
+export const StyledModal = () => (
+  <Modal
+    disablePrimaryButton={false}
+    onHandledPrimaryAction={action('Primary Action')}
+    primaryButtonName="Danger"
+    primaryButtonType="danger"
+    show
+    onHide={action('hidden')}
+    title="I'm a modal!"
```

Comment:
@riahk I'm not sure if all these props are really needed for the story, or if there are any bells/whistles/variations you might think are warranted. Happy to discuss/enhance.

Reply:
Is it possible to add links to external docs in storybook? Might be cool to give easy access to the AntD modal docs in this case.

Reply:
Hmmm... I don't see why not. The Story can have basically any ol' HTML in it, so some of that could just be a `<p>` tag or whatever